Client/src/Features/UptimeMonitors/uptimeMonitorsSlice.js

```
@@ -31,6 +31,30 @@ export const createUptimeMonitor = createAsyncThunk(
+	"monitors/checkEndpoint",
+	async (data, thunkApi) => {
+		try {
+			const { authToken, monitorURL } = data;
+
+			const res = await networkService.checkEndpointResolution({
+				authToken: authToken,
+				monitorURL: monitorURL,
+			})
+			return res.data;
+		} catch (error) {
+			if (error.response && error.response.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)